packages/components/src/core/ButtonIcon/index.stories.tsx

```diff
@@ -95,7 +95,7 @@ export const Default = {
 // Live Preview
 
 const LivePreviewDemo = (): JSX.Element => {
-  const spacings = defaultAppTheme?.spacing;
+  const spacings = sharedAppTheme?.spacing;
```

Comment:
Probably okay? But there might be a better way to access the theme in the story that we could use without having to import it directly.